docs/manual/job-options.md

```diff
@@ -543,7 +549,25 @@ Name Value List with default selections:
 ]
 ```
 
-### URL connection parameters
+#### Using Nested JSON Elements
```

Comment:
Let's include a link to an official (or popular) JSON Path resource for deeper reference.